implementation/src/main/java/io/smallrye/mutiny/groups/UniCreate.java

```diff
@@ -263,8 +262,9 @@ private UniCreate() {
      *
      * @return the new {@link Uni} with a {@code null} item
      */
+    @SuppressWarnings("unchecked")
     public Uni<Void> voidItem() {
-        return UNI_OF_VOID;
+        return UNI_OF_NULL;
```

Comment:
You could avoid the extra unchecked by delegating directly to `nullItem`.  *That* should be trivially inferrable at least...

Reply:
Doh, you're right.

Running a local build to verify we're still good

Reply:
. o O { Hmm, how can I be a royal pain to Ken today... }

Reply:
Gives me something to do when the meetings i'm in don't directly involve me